common/lib/xmodule/xmodule/video_module/video_handlers.py

```diff
@@ -276,7 +276,7 @@ def transcript(self, request, dispatch):
 
         elif dispatch.startswith('available_translations'):
 
-            available_translations = self.available_translations(transcripts)
+            available_translations = self.available_translations(transcripts, verify_assets=True)
```

Comment:
This used to be `verify=True` implicitly, now it's explicit since the signature of `available_translations()` have changed.